python/tvm/topi/hexagon/conv2d.py

```diff
@@ -0,0 +1,26 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+""" Schedules for conv2d. """
+
+import tvm
+
+
+def schedule_conv2d_nhwc(outs):
+    """Schedule for Conv2d NHWC operator."""
+    s = tvm.te.create_schedule([x.op for x in outs])
```

Comment:
For Hexagon, don't we need mark `pipeline` attribute no longer anymore?

Reply:
We're moving towards running entire graphs on Hexagon, so no offloading is necessary.  You can still write an op that will use `pipeline`, but for relay-based compilation we will use ops that run entirely on Hexagon.

Reply:
Can we add a possibility to appoint a layer to be offloaded to CPU? I have compared TFLite case and SNPE one for object detection models and it happened that it's better to execute Detection Output on CPU instead of DSP. It might be an exception but such use case is quite widely used

Reply:
Qualcomm ships products that do this sort of a thing, but I'm not familiar with how the separation between CPU and DSP happens there.  In these products, the communication between CPU and DSP is handled by the runtime that is a part of these products, and the offloading does not occur within an individual op.  That means that even there there is no need to offload anything explicitly in TVM, because the entire op will run either on CPU or on DSP.

Reply:
I might have misunderstood your message `We're moving towards running entire graphs on Hexagon, so no offloading is necessary.` My point that sometimes it might be more efficient to offload some parts and give control to user and I got impression that we are going to cancel this ability.

Reply:
Yes, but we're not planning to support this in the near term.